bedrock/externalpages/templates/externalpages/pocket/add.html

```diff
@@ -18,26 +20,21 @@
 
 <div class="add-page">
    <section class="add-header">
-      <div class="add-header__wrapper">
-         <div class="add-header__title">
-            <h2>Save to Pocket from Firefox</h2>
-         </div>
-         <div class="add-header__context">
-            <div class="add-header__context-image"><img src="{{ static('img/externalpages/pocket/add-pocket-header.jpg') }}" alt="Pocket logo in browser taskbar"></div>
-            <div class="add-header__context-info">
-               <p>
-                  Click the Pocket button in your Firefox toolbar to save articles, videos,
-                  pages, and more.
-               </p>
-               <a href="https://help.getpocket.com/article/900-saving-to-pocket-and-viewing-your-list-in-firefox" class="btn red-btn" target="_blank" rel="noopener noreferrer">Learn How</a>
-               <p>
-                  <a href="https://help.getpocket.com/article/942-where-is-the-pocket-button-in-firefox" class="dark-inline-link rarrow" target="_blank" rel="noopener noreferrer">
-                    Don't see the Pocket button?
-                  </a>
-               </p>
-            </div>
-         </div>
-      </div>
+         {% call split(
+          block_class='mzp-l-split-center-on-sm-md mzp-t-content-xl mzp-l-split-reversed mzp-l-split-body-narrow',
+          media_class='mzp-l-split-h-center',
+          image_url='img/externalpages/pocket/add-pocket-header.jpg',
+          )%}
+          <h2>Save to Pocket from Firefox</h2>
+          <p>Click the Pocket button in your Firefox toolbar to save articles, videos, pages, and more.</p>
+          <a href="https://help.getpocket.com/article/900-saving-to-pocket-and-viewing-your-list-in-firefox" class="mzp-c-button" target="_blank" rel="external noopener">Learn how</a>
+           <p>
+              <a href="https://help.getpocket.com/article/942-where-is-the-pocket-button-in-firefox" class="dark-inline-link rarrow" target="_blank" rel="noopener noreferrer">
+                Don't see the Pocket button?
+              </a>
+          </p>
+          {% endcall %}
```

Comment:
While I love that we're using split here, the layout seems to be a little different from what is on production. I'm going to check in with Dan to make sure he doesn't see any issue with changing it.

Reply:
Actually just talked with dan and he feels like its fine! So we can keep the split!

Reply:
@nathan-barrett glad to hear, the responsiveness of Split was easier to handle for this page than if we didn't use it :)